lib/utils.js

```diff
@@ -13,9 +13,13 @@ exports.emitErr = function (stream, err) {
 // and all matched files. Switching options does not break Sass cache so we do
 // it ourselves. Possibly a bug: https://github.com/sass/sass/issues/1830
 exports.createIntermediatePath = function (sources, matches, options) {
+	defaults.tempSubDir = 'gulp-ruby-sass';
```

Comment:
The `defaults` object is part of this plugin: https://github.com/lkmorlan/gulp-ruby-sass/blob/master/index.js#L22-L27

```js
var uid = typeof process.getuid === 'function' ? process.getuid() : '';

var defaults = {
	tempDir: osTmpdir() + uid,
	...
```